services/blockchain-connector/methods/blocks.js

```diff
@@ -83,4 +85,28 @@ module.exports = [
 		controller: getGenesisConfig,
 		params: {},
 	},
+	{
+		name: 'getGenesisAssetByModule',
+		controller: async ({ module, subStore, limit, offset }) => getGenesisAssetByModule({
+			module,
+			subStore,
+			limit,
+			offset,
+		},
+		),
+		params: {
+			module: { type: 'string' },
```

Comment:
```suggestion
			module: { type: 'string', optional: true },
```

Reply:
If we do not require the module, then we would fetch data for all module which could lead to the time out issue. That is why module is a required parameter here.

Reply:
In that case, can we explicitly set `optional` to false
```
module: { type: 'string', optional: false },
```

Reply:
params are required by default, that is why removed the explicit declaration after adding.